endoreg_db/services/video_import.py

```
@@ -4,19 +4,18 @@
 Provides high-level functions for importing and anonymizing video files,
 combining VideoFile creation with frame-level anonymization.
 """
-
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)